hack/jenkins/e2e.sh

```diff
@@ -748,7 +748,7 @@ case ${JOB_NAME} in
     : ${E2E_TEST:="false"}
     : ${E2E_DOWN:="false"}
     : ${ENABLE_EXPERIMENTAL_API:=true}
-    : ${NUM_MINIONS:=3}
+    : ${NUM_MINIONS:=5}
```

Comment:
Seeing as this number is consistent across a large number of upgrade jobs, please factor it out into a common constant
'''
NUM_MINIONS_UPGRADES=5
...
kubernetes-upgrade-gce-step1-deploy)
...
: NUM_MINIONS:=NUM_MINIONS_UPGRADES
...
'''
etc.